lib/cbus_elixir/app/speaker.ex

```diff
@@ -10,6 +10,7 @@ defmodule CbusElixir.App.Speaker do
     field :name, :string
     field :title, :string
     field :url, :string
+    has_many :speakers, CbusElixir.App.Speaker
```

Comment:
I think this is incorrect? A speaker has many speakers?

Reply:
Good catch.

Reply:
Updated and pushed.